docs/how-to/analytics-dashboard.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Analytics Dashboard
+sidebar_label: Analytics Dashboard
+slug: /analytics-dashboard
+---
+
+# Analytics Dashboard
+
+Amplication's Analytics Dashboard is a powerful tool that provides developers with valuable insights into their application's development process.
+
+The dashboard highlights key metrics related to productivity, efficiency, and code quality to help you understand the impact of using Amplication on your development workflow and the overall return on investment.
```

Comment:
Let's avoid talking explicitly on "return on investment" - it sounds a little apologetic, and we don't really want to initiate such discussion (explicitly)